content/2015.11.shweta-oak.rst

```diff
@@ -0,0 +1,80 @@
+Discovery of Kinto servers
+###########################
+
+:slug: outreachy-2016
```

Comment:
The slug should be unique. I think you should prefix them with "outreachy" and have it named something like "outreachy-kinto-instance-discovery"